.coveragerc

```diff
@@ -300,7 +300,7 @@ omit =
     homeassistant/components/freebox/sensor.py
     homeassistant/components/freebox/switch.py
     homeassistant/components/fritz/device_tracker.py
-    homeassistant/components/fritzbox_callmonitor/sensor.py
+    homeassistant/components/fritzbox_callmonitor/*
```

Comment:
We should revert this change. Add `__init__.py` if we don't want to cover that. With this change we won't capture the config flow coverage.

Reply:
Added a check in #41876 to catch these automatically :)